<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignupWindow</class>
 <widget class="QMainWindow" name="SignupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ﾄ斉ハg kﾃｽ</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
     QWidget{
        background:#0f172a;
        color:white;
        font:12pt &quot;Segoe UI&quot;;
     }
     QLineEdit{
        background:#020617;
        border:1px solid #334155;
        border-radius:6px;
        padding:8px;
        color:white;
     }
     QPushButton{
        border-radius:6px;
        padding:10px;
     }
     #signup{
        background:#2563eb;
     }
     #had{
        background:#111827;
        border:1px solid #374151;
     }
    </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="credit_btn">
      <property name="styleSheet">
       <string notr="true">#credit_btn{
background:#2563eb;
border-radius:6px;
padding:10px;}
#credit_btn:hover{
background:rgb(29, 80, 189)
}
</string>
      </property>
      <property name="text">
       <string>Credit</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>120</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string>font-size:40px;</string>
        </property>
        <property name="text">
         <string>洫</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string>font-size:22px;font-weight:600;</string>
        </property>
        <property name="text">
         <string>ﾄ斉ハg kﾃｽ</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string>color:#9ca3af;</string>
        </property>
        <property name="text">
         <string>T蘯｡o tﾃi kho蘯｣n vﾃ qu蘯｣n lﾃｭ l盻孅 h盻皇</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>420</width>
          <height>320</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>450</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
           QFrame{
            background:#020617;
            border:1px solid #1f2937;
            border-radius:12px;
           }
QPushButton{
background-color:#0F172A;
border-radius:4px;
border: 1px solid #374151;
}
QPushButton:hover{
background-color:rgb(38, 59, 107);
}
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Tﾃｪn giﾃ｡o viﾃｪn</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name1">
           <property name="placeholderText">
            <string>Vﾃｭ d盻･: Lﾃｲ Vﾄハ Tﾃｹng</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>M蘯ｭt kh蘯ｩu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pass1">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Xﾃ｡c nh蘯ｭn m蘯ｭt kh蘯ｩu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="pass_confirm">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="signup">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>ﾄ斉ハg kﾃｽ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="had">
           <property name="text">
            <string>ﾄ静｣ cﾃｳ tﾃi kho蘯｣n? ﾄ斉ハg nh蘯ｭp</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
